paasta_tools/kubernetes_tools.py

```diff
@@ -1679,6 +1689,9 @@ def get_pod_template_spec(
             # character limit for k8s labels (63 chars)
             labels["paasta.yelp.com/deploy_group"] = self.get_deploy_group()
 
+        if piscina_autoscaling_configured:
+           labels["paasta.yelp.com/scrape_piscina_prometheus"] = "true"
```

Comment:
can we also add the label from https://github.com/Yelp/paasta/pull/3235/files#diff-f1b278fb2675abadd2f58f131c3f67ece5d7511b60fcad691255aeda6f490d17R1690?

this really should be a default label, but we forgot to promote this and I don't think we want to cause a big bounce for code that we're just rolling out for the first time